engine/util/src/main/scala/pl/touk/nussknacker/engine/util/json/BestEffortJsonEncoder.scala

```diff
@@ -50,6 +52,7 @@ case class BestEffortJsonEncoder(failOnUnkown: Boolean, highPriority: PartialFun
       case a: scala.collection.Map[String@unchecked, _] => encodeMap(a.toMap)
       case a: java.util.Map[String@unchecked, _] => encodeMap(a.asScala.toMap)
       case a: Traversable[_] => fromValues(a.map(encode).toList)
+      case a: Enum[_] => a.getClass.getEnumConstants.collectFirst { case c if c.equals(a) => c }.map(c => safeString(c.toString)).get
```

Comment:
maybe I'm stupid but it is not the same as just `a.toString` :) ? Can you add unit test to  `BestEffortJsonEncoderSpec` ?